pkg/virtualservice/utils.go

```diff
@@ -0,0 +1,17 @@
+package virtualservice
+
+import (
+	appmesh "github.com/aws/aws-app-mesh-controller-for-k8s/apis/appmesh/v1beta2"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// IsVirtualServiceActive checks whether given virtualService is active.
+// virtualService is active when its VirtualServiceActive condition equals true.
+func IsVirtualServiceActive(vs *appmesh.VirtualService) bool {
```

Comment:
Is this used anywhere except in the test?

Reply:
no, it's not used anywhere. (but will be used in future PRs) :D